components/enedis/enedis.app.mjs

```diff
@@ -0,0 +1,123 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "enedis",
+  propDefinitions: {
+    start: {
+      label: "Start",
+      description: "Start date of the period in RFC-3339 format (YYYY-MM-DD) e.g. '2022-10-28'. The returned datas will include the start date.",
+      type: "string",
+    },
+    end: {
+      label: "End",
+      description: "End date of the period in RFC-3339 format (YYYY-MM-DD) e.g. '2022-11-01'. The returned datas will not include the end date.",
+      type: "string",
+    },
+    usage_point_id: {
+      label: "Usage point id",
+      description: "Usage point id known by the user. Use '12655648759651' if don't have one.",
+      type: "string",
+    },
+  },
+  methods: {
+    _getUrl(path) {
+      return `https://ext.hml.api.enedis.fr${path}`;
+    },
+    _getHeaders() {
+      return {
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+		"User-Agent": "@PipedreamHQ/pipedream v0.1",
+      };
+	},
+    prepareParam(self) {
+      return {
+        "usage_point_id": self.usage_point_id,
+      };
+    },
+    prepareAllParams(self) {
+      return {
+        "usage_point_id": self.usage_point_id,
+        "start": self.start,
+        "end": self.end,
+      };
+    },
+    async _makeRequest({
+      $,
+      path,
+	  params
+    } = {}) {
+      const config = {
+        url: this._getUrl(path),
```

Comment:
Personally I like to define a `baseURL` property, and accept a `url` param from the methods - Axios then internally concatenates the strings as you did here.

```js
```
```suggestion
        baseURL: this._getUrl(), // this method would return just the base URL
        url: path, // 'path' could be received as 'url' instead
```

This is just a personal preference though - totally up to you if you want to give it a shot.